internal/strategies/gradualrolloutuserid_test.go

```diff
@@ -0,0 +1,112 @@
+package strategies
+
+import (
+	"github.com/unleash/unleash-client-go/context"
+	"github.com/unleash/unleash-client-go/strategy"
+	"strconv"
+	"testing"
+)
+
+func TestGradualRolloutUserId_Name(t *testing.T) {
```

Comment:
@ivarconr For local paths, you need to start with `./` otherwise it assumes you are referring to a top level package in which you need the fully qualified path:

`go test github.com/unleash/unleash-client-go/internal/...`